src/schema/imported/manifest.json

```diff
@@ -104,7 +104,7 @@
                   "enum": [
                     "spanning"
                   ],
-                  "onError": "warn"
+                  "default": "spanning"
```

Comment:
Just as a side note, this change means that extensions with "not_allowed" or "split" as the requested incognito mode will start to get errors when submitted on AMO.

The error message we currently report for this validation error is unfortunately not super useful, as it doesn't currently say which are the allowed values (currently just "spanning"):
```
Code           Message                           Description                           File   Line   Column
JSON_INVALID   "/incognito" should be equal to   Your JSON file could not be parsed.
               one of the allowed values
```

Shane checked with Mike how many of such extension we have, and they should be very few (I recall there were less then 10).

Anyway, Firefox will also raise an error an refuse to install the extension, and so the linter and AMO behavior is going to be consistent with Firefox.